build/ci/windows/setup.bat

```diff
@@ -38,13 +38,13 @@ ECHO "=== Install Qt ==="
 SET "Qt_ARCHIVE=Qt5152_msvc2019_64.7z"
 
 IF %TARGET_PROCESSOR_BITS% == 32 (
-    ::SET "Qt_ARCHIVE=qt599_msvc2015.7z"
-    ECHO "error: Not avalable Qt 32"
+    SET "Qt_ARCHIVE=Qt5152_msvc2019.7z"
+    ECHO "Error: Qt 32-bit (%Qt_ARCHIVE%) not (yet) available"
```

Comment:
Something to take care of for Alpha or latest for Beta, RC and Release, which should come as a 32bit version too.